### PR TITLE
feat: tds scoring and api implementation

### PR DESCRIPTION
**Pull request by sunnygleason**
Problem:

* the current bikes don't race 😞 

Solution:

* implement the race position function as: `blockBasedPosition + stakeBasedAdjustment` 🚲 
* since it's so hard to test, put in a demo mode for the meantime so folks can check out the mechanics 🐎

### CODE REVIEW

**Comment by sunnygleason** [comment]
* @CriesofCarrots this is ready for review
* @manuel-calavera FYI, this includes your suggestion to move logic into API, feedback welcome

**Comment by sunnygleason** [comment]
(ugh commit  lint)

**Comment by manuel-calavera** [comment]
@sunnygleason thanks for moving it!

File: src/v2/components/TourDeSol/Cards/index.jsx [hunk 1]
```diff
@@ -53,31 +54,32 @@ const Cards = ({
     },
     {
       title: 'Circulating SOL',
-      value: (supply / Math.pow(2, 34)).toFixed(2),
+      value: totalSupply && totalSupply.toFixed(2),
```
Comment (CriesofCarrots) [inline src/v2/components/TourDeSol/Cards/index.jsx]: 👍

File: src/v2/constants.js [hunk 2]
```diff
@@ -1,33 +1,4 @@
 export const MIN_TERM_LEN = 3;
 export const LAMPORT_SOL_RATIO = 0.0000000000582;
```
[the remark this answers is not in the document]
Comment (sunnygleason) [reply src/v2/constants.js]: 👍; the new view models return SOLs from the API; I'll add a cleanup ticket to track this & other v2 cleanup

[the remark this answers is not in the document]
Comment (sunnygleason) [reply src/v2/constants.js]: added #531

**Comment by sunnygleason** [comment]
FYI this requires a JS client library update to become visible on edge w/ the `rootSlot` change

**Pull request merged**